design-editor/src/system/stage-manager.js

```diff
@@ -237,7 +239,8 @@ class StageManager {
      */
     _togglePreview(toggleEditor) {
         var $workSpace = $(editor.selectors.workspace),
-            preview = null;
+			preview = null,
+			isDemoVersion = PreferencesManager.getViewState('projectType') === 'demo' ? true : false;
```

Comment:
to stay DRY I'd create a util file in utils catalog and add there a function isDemoVersion which would be used both here and in design-editor-element.